includes/application/forwardApprover.php

```diff
@@ -63,20 +63,38 @@
 	if( isset( $_SESSION['PostName'] ) )
 	{
 		$UID = $_SESSION['PostName'];
+		if ( forwardApprover( $app_id ,$app_type, $UID ) == true )
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 0 spaces after opening bracket; 1 found
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Space found before comma in function call
* No space found after comma in function call
* Expected 1 space after closing parenthesis; found 9

<!-- Comment by [SideCI](https://sideci.com) -->